commands/new_function_test.go

```diff
@@ -118,13 +122,36 @@ var NewFunctionTests = []NewFunctionTest{
 		funcLang:    "",
 		expectedMsg: NoLanguage,
 	},
+	{
+		title:         "version-with-white-space",
+		funcName:      "version-with-white-space",
+		funcLang:      "ruby",
+		schemaVersion: " ",
+		expectedMsg:   EmptyVersion,
+	},
+	{
+		title:         "invalid-version",
+		funcName:      "invalid-version",
+		funcLang:      "ruby",
+		schemaVersion: "2.0",
+		expectedMsg:   InvalidVersion,
+	},
+	{
+		title:         "valid-version",
+		funcName:      "valid-version",
+		funcLang:      "ruby",
+		expectedImage: "valid-version:latest",
```

Comment:
Let's add a few more test cases - valid with a leading space, valid with a trailing space, valid with both.

Reply:
I removed the flag and the associated tests along with it.